topics/admin/tutorials/external-auth/tutorial.md

```diff
@@ -98,7 +99,7 @@ For this exercise we will use a basic password file method for authenticating -
 >      galaxy:
 >        ...
 >        use_remote_user: true
->        remote_user_maildomain: "{{ hostname }}"
+>        remote_user_maildomain: "{% raw %}{{ hostname }}{% endraw %}"
```

Comment:
does this produce a full enough hostname to work? With the .eu? Or just `gcc-8` or so?

Reply:
```suggestion
>        remote_user_maildomain: "{% raw %}{{ inventory_hostname }}{% endraw %}"
```